app/konnect/runtime-manager/kic.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Kubernetes Ingress Controller Integration
+content_type: explanation
+---
+
+You can use native Kubernetes resources to configure your clusters in Konnect by integrating your KIC deployment with Konnect. This set up is ideal for organizations who want to manage gateways in Konnect through native Kubernetes resources without having to use a hybrid deployment model. This guide explains how to add a KIC runtime group to {{site.konnect_short_name}} and the benefits of this integration. 
+
+## About KIC in Konnect
+
+Kubernetes Ingress Controller (KIC) for Kong configures {{site.base_gateway}} using Ingress or [Gateway API](https://gateway-api.sigs.k8s.io/) resources created inside a Kubernetes cluster. KIC for Kong performs more than just proxying the traffic coming into a Kubernetes cluster. You can also configure plugins, load balancing, health checking, and leverage all that {{site.base_gateway}} offers in a standalone installation. For more information about KIC for Kong, see [Kubernetes Ingress Controller Design](/kubernetes-ingress-controller/latest/concepts/design/). 
+
+By integrating your KIC deployment with Konnect, this read-only integration allows you to view your runtime entities, such as routes and applications, from your Kubernetes resources in Konnect.  
+
+Here are a few benefits to adding a KIC runtime group to Konnect instead of using KIC on its own:
+* **Easy to set up:** The wizard allows you to add your KIC runtime group to Konnect in minutes.
+* **Monitor your KIC analytics:** By integrating with Konnect, you can view the analytics from your KIC runtime instances alongside any of your self-managed Konnect runtime instances. 
+* **Display KIC entities in Dev Portal:** Publish your KIC services to the Dev Portal and make the API specs available to third-party developers. 
+
+## KIC in Konnect integration
+
+To integrate your runtime instances in KIC with Konnect, you can use the wizard in the UI to add your KIC deployment to Konnect by navigating to **[Runtime Manager](https://cloud.konghq.com/runtime-manager) > New Runtime Group > Kubernetes Ingress Controller**.
+
+{:.note}
+> **Note**: KIC OSS users can connect to Konnect’s Free tier, while KIC Enterprise users can connect to Konnect’s Enterprise tier.
+
+### Prerequisites
+
+If you don't have an existing KIC deployment, you need the following before using the instructions in the wizard:
+*  A Kubernetes cluster 
+* `kubectl` or `oc` (if you're working with OpenShift) installed and configured to communicate with your Kubernetes TLS
+* [Helm 3](https://helm.sh/docs/intro/install/) installed
+
+### View KIC entities
+
+After your KIC deployment is connected to Konnect, you can view the details for each runtime instance in your KIC runtime groups. 
+
+[SCREENSHOT]
+
+Table with things from screenshot of dashboard?
```

Comment:
❗ Reviewers: I thought we could add a table that describes the parts of the screenshot here, though I'm not sure how useful that would be since I think the UI is pretty intuitive here.

Reply:
Answered in parallel screenshot PR.